sky/execution.py

```diff
@@ -210,6 +210,11 @@ def _execute(
             if not down:
                 requested_features.add(
                     clouds.CloudImplementationFeatures.AUTOSTOP)
+                # TODO(suquark): enable GCP+spot to be stopped in the future.
```

Comment:
Seems not needed?

Reply:
I was under the impression that if we support GCP+spot stop, then we would support `sky launch --cloud gcp --use-spot -i 5`?

Reply:
Yes, then should we change it to `TODO: allow autostop for spot when stopping is supported.`

Reply:
Ah, got it. That makes sense; I will fix

Reply:
Updated!